injected/src/features/message-bridge.js

```diff
@@ -0,0 +1,168 @@
+import { Messaging } from '@duckduckgo/messaging';
+import ContentFeature from '../content-feature';
+import {
+    InstallProxy,
+    ProxyNotification,
+    ProxyRequest,
+    ProxyResponse,
+    SubscriptionRequest,
+    SubscriptionResponse,
+    SubscriptionUnsubscribe,
+} from './message-bridge/schema.js';
+
+/**
+ * This part has access to messaging handlers
+ */
+export class MessageBridge extends ContentFeature {
+    proxies = new Map();
+    subscriptions = new Map();
+
+    init(args) {
+        console.log(`[isolated] will install bridge with ${args.sessionKey}`);
+        /**
+         * TODO: what is the value of this token? - since 'createPageWorldBridge' is available to all page scripts, probably none
+         * @param {string} eventName
+         * @return {`${string}-${string}`}
+         */
+        function appendToken(eventName) {
+            return `${eventName}-${args.sessionKey}`;
+        }
+        window.addEventListener(appendToken(InstallProxy.NAME), (/** @type {CustomEvent<unknown>} */ e) => {
+            console.log(`[isolated:incoming] ${InstallProxy.NAME}`, e.detail);
+            const incoming = InstallProxy.create(e.detail);
+            if (!incoming) return;
+            this.installProxyFor(incoming.featureName, args.messagingConfig);
+        });
+        window.addEventListener(appendToken(ProxyNotification.NAME), (/** @type {CustomEvent<unknown>} */ e) => {
+            console.log(`[isolated:incoming] ${ProxyNotification.NAME}`, e.detail);
+            const notification = ProxyNotification.create(e.detail);
+            if (!notification) return;
+            this.proxyNotification(notification);
+        });
+        window.addEventListener(appendToken(ProxyRequest.NAME), (/** @type {CustomEvent<unknown>} */ e) => {
+            console.log(`[isolated:incoming] ${ProxyRequest.NAME}`, e.detail);
+            const request = ProxyRequest.create(e.detail);
+            if (!request) return;
+            this.proxyRequest(request, appendToken);
+        });
+        window.addEventListener(appendToken(SubscriptionRequest.NAME), (/** @type {CustomEvent<unknown>} */ e) => {
+            console.log(`[isolated:incoming] ${SubscriptionRequest.NAME}`, e.detail);
```

Comment:
This debugging feels like it might be useful, can you make a file specific debug method that is disabled by default?